generate-component-docs/__snapshots__/generate.test.ts.snap

```diff
@@ -0,0 +1,6618 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Public API Actions 1`] = `
+Object {
+  "props": Object {
+    "children": Object {
+      "propName": "children",
+      "required": true,
+      "type": Object {
+        "type": "union",
+        "types": Array [
+          "string",
+          "number",
+          "false",
+          "true",
+          "{}",
+          "ReactElement<any, string | ((props: any) => ReactElement<any, string | ... | (new (props: any) => Component<any, any, any>)> | null) | (new (props: any) => Component<any, any, any>)>",
+          "ReactNodeArray",
+          "ReactPortal",
+        ],
+      },
+    },
+  },
+}
+`;
+
+exports[`Public API Alert 1`] = `
+Object {
+  "props": Object {
+    "children": Object {
+      "propName": "children",
+      "required": true,
+      "type": Object {
+        "type": "union",
+        "types": Array [
+          "string",
+          "number",
+          "false",
+          "true",
+          "{}",
+          "ReactElement<any, string | ((props: any) => ReactElement<any, string | ... | (new (props: any) => Component<any, any, any>)> | null) | (new (props: any) => Component<any, any, any>)>",
+          "ReactNodeArray",
+          "ReactPortal",
+        ],
+      },
+    },
+    "tone": Object {
+      "propName": "tone",
+      "required": false,
+      "type": Object {
+        "type": "union",
+        "types": Array [
+          "\\"info\\"",
+          "\\"critical\\"",
+          "\\"positive\\"",
+        ],
+      },
+    },
+    "weight": Object {
+      "propName": "weight",
+      "required": false,
+      "type": Object {
+        "type": "union",
+        "types": Array [
+          "\\"strong\\"",
+          "\\"regular\\"",
+        ],
+      },
+    },
+  },
+}
+`;
+
+exports[`Public API Box 1`] = `
+Object {
+  "props": Object {
+    "about": Object {
+      "propName": "about",
```

Comment:
Any thoughts on being able to apply the breaks on components like this and not traverse every html prop?

Reply:
I thought about it. It's probably doable. But it's still part of our contract. Was thinking we could see how it goes.